blivet/devicelibs/stratis.py

```
@@ -253,14 +253,21 @@ def create_pool(name, devices, encrypted, passphrase, key_file, clevis):
     except DBusError as e:
         raise StratisError("Failed to create stratis pool: %s" % str(e))
     else:
         if not succ:
             raise StratisError("Failed to create stratis pool: %s (%d)" % (err, rc))
 
+    try:
+        proxy = util.SystemBus.get_proxy(STRATIS_SERVICE, pool_path,
+                                         STRATIS_POOL_INTF)
+        proxy.Overprovisioning = overprovisioning
+    except DBusError as e:
+        raise StratisError("Failed to enable overprovisioning on stratis pool: %s" % str(e))
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)